docs/source/specs/typespec/main.tsp

```diff
@@ -809,9 +768,6 @@ namespace RoleBindings {
         @doc("Cursor for cursor-based pagination.")
         @query cursor?: string;
 
-        @doc("Filter by resource type")
-        @example("workspace")
-        @query resource_type?: string;
```

Comment:
Why this is removed? If in the future there are a lot of resource types, it is hard to determine the resource only by id, right?